assets/model_monitoring/components/src/model_monitor_compute_histogram_buckets/histogram_buckets.py

```diff
@@ -24,6 +24,8 @@ def compute_numerical_bins(
     # Generate histograms only for columns in both baseline and target dataset
     common_columns_dict = get_common_columns(df1, df2)
     numerical_columns = get_numerical_cols_with_df(common_columns_dict, df1)
+    if numerical_columns is None or len(numerical_columns) <= 0:
+        raise ValueError("Could not find common columns for compute histogram buckets.")
```

Comment:
Can we define an error Type for this kind of validation.

Richard defined the nodatainput, but it's not an error hierarchy, I am thinking is it possible to define a baseErrorType Like UserError. And this can be UserError/NoCommonColumns? So that in dashboard we can track it.
Any options to achieve that?